rabbit-sink/src/main/java/org/springframework/cloud/stream/module/rabbit/sink/RabbitSinkProperties.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.rabbit.sink;
+
+import javax.validation.constraints.AssertTrue;
+
+import org.springframework.amqp.core.MessageDeliveryMode;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.expression.Expression;
+
+/**
+ * Properties for the Rabbit Sink.
+ *
+ * @author Gary Russell
+ *
+ */
+@ConfigurationProperties
+public class RabbitSinkProperties {
+
+	public static final String JSON_CONVERTER = "jsonConverter";
+
+	/**
+	 * Exchange name - overridden by exchangeNameExpression, if supplied.
+	 */
+	private String exchange = "";
+
+	/**
+	 * A SpEL expression that evaluates to an exchange name.
+	 */
+	private Expression exchangeExpression;
+
+	/**
+	 * Routing key - overridden by routingKeyExpression, if supplied.
+	 */
+	private String routingKey;
+
+	/**
+	 * A SpEL expression that evaluates to a routing key.
+	 */
+	private Expression routingKeyExpression;
+
+	/**
+	 * Default delivery mode when 'amqp_deliveryMode' header is not present.
+	 */
+	private MessageDeliveryMode deliveryMode = MessageDeliveryMode.NON_PERSISTENT;
+
+	/**
+	 * Headers that will be mapped.
+	 */
+	private String mappedRequestHeaders[] = { "*" };
```

Comment:
Well, with the `negated patterns` I'm sure we can change the default for the AMQP header mapping to the all since SI-4.3, like it is with JMS.
And may review all others which supports `negated patterns`, too...